### ISSUE TITLE
Questions about the nn.ConvTranspose2d in features_mask

### ISSUE THREAD

**Issue by city19992** [EastbeanZhang/Gaussian-Wild#18]
Great job!

While I have a question about the mask decoder (lines 112 to 121 of [feature_maps_generators.py](https://github.com/EastbeanZhang/Gaussian-Wild/blob/main/net_modules/feature_maps_generators.py)).
You used the` nn.ConvTranspose2d` for upsampling. I wonder why you used this solution for upsampling. Have you ever tried other ways like pixelshuffle?

**Comment by EastbeanZhang** [comment]
We experimented with ConvTranspose2d and linear upsampling + Conv2d. The difference between them was negligible. If you're interested, you could try pixelshuffle.

**Comment by city19992** [comment]
Thank you so much!
